presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -326,48 +316,6 @@ public void testTableRulesForCheckCanRenameTable()
         assertAccessDenied(() -> accessControl.checkCanRenameTable(ALICE, new CatalogSchemaTableName("some-catalog", "aliceschema", "alicetable"), new CatalogSchemaTableName("some-catalog", "bobschema", "newalicetable")), RENAME_TABLE_ACCESS_DENIED_MESSAGE);
     }
 
-    @Test
-    public void testCanSetUserOperations()
```

Comment:
Don't we need to port this to `testCanImpersonateUser`? I don't see this test here. Maybe it is not needed as it is already tested elsewhere.

Reply:
`testCanImpersonateUserOperations` seems to be complete